### PR TITLE
Remove admin role

### PR DESCRIPTION
**Pull request by otlaitil**
Also removes an email notification / reminder that was sent for admin users.
The functionality was no longer needed

Removes a separate view for internal response in admin
view. This was for admin users. All the changes to petitions
are now done from edit response -view.

Renames threshold role to moderator

### CODE REVIEW

File: spec/factories.rb [hunk 1]
```diff
@@ -15,8 +15,8 @@
     role "sysadmin"
   end
 
-  factory :threshold_user, :parent => :admin_user do
-    role "threshold"
+  factory :moderator_user, :parent => :admin_user do
```
Comment (otlaitil) [inline spec/factories.rb]: This moderator factory is probably redundant. Should I declare admin factory with

```factory :admin, aliases: [:moderator] do```

and get rid of this?

Comment (h-lame) [inline spec/factories.rb]: Maybe just take the role out of the admin_user factory, and make sure we always use sysadmin_user, or moderator_user ?

Comment (otlaitil) [inline spec/factories.rb]: That's better. Thanks.

**Comment by otlaitil** [comment]
PR comments addressed.

**Comment by h-lame** [comment]
We could merge this now I think.

**Comment by davidbasalla** [comment]
:+1:

**Pull request merged**